Tests/DummyData/DummyPaymentMethods.swift

```diff
@@ -102,15 +102,15 @@ let issuerListDictionary = [
         [
             "items": [
                 [
-                    "id": "1121",
+                    "id": "xxxx",
```

Comment:
Is this needed to hide icons?

Reply:
As discussed: need to add test for image loading

Reply:
so that the icon loading fails predictably :-)

Reply:
will add tests in  separate PR